source/simple_socketio.ts

```diff
@@ -140,11 +140,9 @@ export default class SimpleSocketIOClient {
     try {
       const url = new URL(`${this.serverUrl}/socket.io/1/`);
       url.searchParams.append("api_user", this.apiUser);
+      url.searchParams.append("api_key", this.apiKey);
```

Comment:
I would like to avoid exposing this as a query param if we can keep it in the header. Does the server not allow them in Access-Control-Allow-Headers?